tests/syft/core/node/node_test.py

```diff
@@ -209,20 +209,3 @@ def test_send_message_from_network_client_to_vm() -> None:
     bob_network_client.send_immediate_msg_without_reply(
         msg=sy.ReprMessage(address=bob_vm.address)
     )
-
-
-@pytest.mark.slow
-@pytest.mark.asyncio
-def test_autoapprove_requests_made_by_root_clients_5015(
```

Comment:
It should also approve the requests made by simple clients to be consistent with the case when we create an object and simply "get" it without approval?
Or should we change the behavior for "get"?